packages/connect-agreement/subgraph/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Not directly related here, but I saw this in a few other subgraphs too and I'm wondering if we should remove it? It seems weird this would be MIT and copyright with The Graph.

Reply:
I agree I'll create a new PR once we merge this one to remove it everywhere.